myria/connection.py

```diff
@@ -239,13 +239,15 @@ def dataset(self, relation_key):
             relation_key['programName'],
             relation_key['relationName']))
 
-    def download_dataset(self, relation_key):
+    def download_dataset(self, relation_key, limit=None):
         """Download the data in the dataset as json"""
         return self._wrap_get('/dataset/user-{}/program-{}/relation-{}/data'
                               .format(relation_key['userName'],
                                       relation_key['programName'],
                                       relation_key['relationName']),
-                              params={'format': 'json'})
+                              params={'format': 'json',
+                                      'limit': limit if limit is not None
```

Comment:
Can we just omit the `limit` arg if it's `None`? What's the semantics if we pass an empty string?